Source/Util/FlushProcess.swift

```diff
@@ -46,9 +46,15 @@ internal class FlushProcess {
 		
 		DispatchQueue.global(qos: .background)
 			.asyncAfter(deadline: time) { [weak self] in
-			self?.sendEvents()
+			guard let `self` = self else {
```

Comment:
Be careful with using this, its a compiler bug [https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20160118/007426.html](link)

Reply:
nice, i just delete this commit... because sendEvents method is synchronous, the flush process only restart when the sendEvents method finish